### DISCUSSION TITLE
Cinnamon and HDMI Audio

### DISCUSSION THREAD

**Discussion by javiermitchell** [Joshua-Riek/ubuntu-rockchip#16] [General]
Hi Joshua
Just ran your image on my Orange Pi 5 and work very well!
I did some GPU speed tests and it was quite high:4552 points
However ( there is always a however), I don't like  - at all - the new Gnome Desktop
Would it possible for you to create an image with Cinammon instead, and I also missing HDMI audio, so it would brilliant to be able to hear sound directly from the Monitor?
If I were to install the Server ( headless) image, would I be able to say,  install Cinnamon via tasksel, or would that break the GPU drivers?
I don't know much about compiling, but If you give me some instructions, I may be able to do it myself

Thanjs

Javier

**Comment by Joshua-Riek** [comment]
I don't include the GPU drivers in the server image, so installing a desktop would not utilize the GPU. I would love to make some changes, so it would be easy to do just that. But I've focused on fixing many bugs facing the gnome desktop image. 

However, you can attempt the [Armbian](https://www.armbian.com/orangepi-5/) approach to installing GPU drivers, but I can only offer limited support as each desktop has unique quirks with the GPU and platform. I personally don't use this method as I have found issues in the past with it and decided to use my own method, it could be better now I'm not sure.

From doing some quick research Cinnamon might not support wayland which may cause problems, ill build my own test image with cinnamon to see what happens, I can't guarantee anything. 

Also for HDMI audio it should be working, go into Settings -> Sound and under Output Device try both Analog Output options, one of them should work, I plan on fixing this once I get the chance.

**Comment by javiermitchell** [comment]
Joshua
Thanks for your quick reply
I'm surprised when you said there are no GPU drivers in your image, however when I run glmark2-es2-wayland it gives me scores consistent with accelerated hardware!
Also chromium plays hd video without a hitch!
I have tried Armbian, but for some reason I cannot make hardware acceleration to work!
Maybe you are right and Wayland is not supported on this desktop 
Regarding HDMI audio, I did try, and only shows the headphones and built in audio, not HDMI
Can you recommend any desktop that looks like KDE - I use Mint cinnamon as my main distro.
Thanks
Javier
--
Sent from myMail for Android Monday, 13 February 2023, 04:43pm +00:00 from Joshua Riek  ***@***.*** :

**Reply by Joshua-Riek** [reply]
I meant that the ubuntu server image has no GPU drivers; the desktop image does. 
I'll take a look at the HDMI audio issue, I think I have a fix. 
Im not sure about other desktops that look like KDE off the top of my head but ill look into it.

**Comment by Joshua-Riek** [comment]
Hey! I just came out with a new release [v1.5](https://github.com/Joshua-Riek/ubuntu-orange-pi5/releases/tag/v1.5) and made a lot of changes regarding the HDMI audio, it should be working out to the box now.
